src/store/actions/auth/auth.js

```diff
@@ -124,10 +124,12 @@ const _addUser = async (dispatch, getState, authenticatedUserData, userType) =>
     const uid = authenticatedUserData.uid;
 
     // Push the authenticated data of the user to firestore
+
+    const collectionRef = userType == "STUDENT" ? 'student' : 'organization';
```

Comment:
As @Chillisaucery has pointed out, the login action needs to separate the userTypes.